### DISCUSSION TITLE
Integration of Daly BMS

### DISCUSSION THREAD

**Discussion by nodurah** [hoylabs/OpenDTU-OnBattery#210] [Ideas]
Hello, 

I really like the fork and the integration of the charge controller. Setting up a small 12V LiFePo4 solar battery myself I was wondering if it would be possible to integrate the popular Daly Smart BMS from the ESP HOME into your project (https://esphome.io/components/sensor/daly_bms.html).  Right now I will use two ESPs for that purpose. One controlling the inverter and the charger and one for the BMS. Just an idea. Keep up the good work!

Cheers, Nodura

**Comment by helgeerbe** [comment]
@nodurah It's a good idea to support other BMS as well. Schould  be a not so hard task. But in the moment, I don't have the time. But if there is someone who wants to do the integration, I can provide support.

**Comment by nodurah** [comment]
@helgeerbe thanks for the quick reply. As time is always a limiting factor I fully understand that it wouldn't materialize overnight. Unfortunately my C++ skills - rusty as they are - are inadequate for the code in this project. 
Cheers, Nodurah
